less/bootstrap.my.less

```diff
@@ -0,0 +1,25 @@
+@import "bootstrap.less";
+
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 300;
+  src: local('Source Sans Pro Light'), local('SourceSansPro-Light'), url(../fonts/source_sans_light.woff) format('woff');
+}
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Source Sans Pro'), local('SourceSansPro-Regular'), url(../fonts/source_sans.woff) format('woff');
+}
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Source Sans Pro Bold'), local('SourceSansPro-Bold'), url(../fonts/source_sans_bold.woff) format('woff');
+}
+@font-family-sans-serif: "Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans-serif;
+
+.tt-suggestions {
```

Comment:
Unfortunately, this doesn't insert the hover/focus states (because they're declared outside of `.dropdown-menu`), but it doesn't look half bad without that gradient, imho.